src/components/App.tsx

```diff
@@ -77,32 +84,64 @@ function App() {
   const handleFontSelectedLeft = (selectedFont: File | null) => {
     setSelectedFontLeft(selectedFont);
     setGoogleFontLeft(null);
+    setGoogleFontName('', 'left');
     handleFontSelected(selectedFont, 'left');
   };
 
   // Handler for when a font is selected on the right side
   const handleFontSelectedRight = (selectedFont: File | null) => {
     setSelectedFontRight(selectedFont);
     setGoogleFontRight(null);
+    setGoogleFontName('', 'right');
     handleFontSelected(selectedFont, 'right');
   };
 
+  const setGoogleFontName = (fontData: string, side: string) => {
+    let fontName = ' ';
+    const parsedCss = postcss.parse(fontData);
+
+    // Find the first font-family value
+    parsedCss.walkAtRules('font-face', (atRule) => {
+      atRule.walkDecls('font-family', (decl) => {
+        fontName = decl.value.replace(/^['"]|['"]$/g, '');
```

Comment:
Something about the regex compromises the reliability check, try:
`fontName = decl.value.replace(/^(?:['"])|(?:['"]$)/g, '');`

(?:) should group the regex together.